src/scales/scale.time.js

```diff
@@ -175,8 +175,25 @@ module.exports = function(Chart) {
 		var ticks = [];
 		if (options.maxTicks) {
 			var stepSize = options.stepSize;
-			ticks.push(options.min !== undefined ? options.min : niceRange.min);
-			var cur = moment(niceRange.min);
+			var startTick = options.min !== undefined ? options.min : niceRange.min;
+			var majorUnitStart = startTick;
+			if (options.majorUnit) {
+				majorUnitStart = moment(startTick).add(1, options.majorUnit).startOf(options.majorUnit);
+			}
+			var startRange = majorUnitStart.valueOf() - startTick;
+			var startFraction = startRange % (interval[options.unit].size * stepSize);
+			var alignedTick = startTick;
+			ticks.push(startTick);
+			if (startTick !== alignedTick + startFraction &&
+				options.majorUnit &&
```

Comment:
I'm used to `&&` and `||` at the end but it's not enforced by the linter.

This code should fit in one line and be more readable:
- `startTick !== alignedTick + startFraction` eq. `!startFraction`
- `majorUnit` should be cached because it's used many time
- same for `timeOpts`

```javascript
var majorUnit = options.majorUnit;
var timeOpts = options.timeOpts;
//...
if (majorUnit  && !startFraction && !timeOpts.round && !timeOpts.isoWeekday) {
    // ...
}
```